client/app/main/visualization/visualization.controller.js

```diff
@@ -2,97 +2,115 @@
+    cfa: { arca: 0, ca: 0, other: 0 }, 
+    nonCfa: { arca: 0, ca: 0, other: 0 } 
+  };
   $scope.chartConfig = {
     options: {
       chart: {
-        zoomType: 'x',
-        type: 'line'
-      }
-    },
-    xAxis: {
-      type: 'datetime',
-      title: {
-        text: 'Date'
-      }
-    },
-    yAxis: {
+        plotBackgroundColor: null,
+        plotBorderWidth: null,
+        plotShadow: false,
+        type: 'pie'
+      },
       title: {
-        text: $scope.truancyType
+        text: 'ARCA vs CA'
       },
-      allowDecimals: $scope.truancyType === 'Average'
-    },
-    title: {
-      text: $scope.statType + ' ' + $scope.truancyType + ' ' + 'Trends'
+      tooltip: {
+        pointFormat: '{series.name}: <b>{point.percentage:.1f}%</b>'
+      },
+      plotOptions: {
+        pie: {
+          allowPointSelect: true,
+          cursor: 'pointer',
+          dataLabels: {
+            enabled: false,
+            format: '<b>{point.name}</b>: {point.percentage:.1f} %',
+            style: {
+                color: (Highcharts.theme && Highcharts.theme.contrastTextColor) || 'black'
+            }
+          },
+          showInLegend: true
+        }
+      }
     },
-    loading: false,
     func: function(chart) {
       $scope.chartObj = chart;
     }
   };
 
-  Visualization.schools().$promise.then(function(results) {
-    $scope.Cumulative = {
-      Absences: [],
-      Tardies: []
-    };
-    $scope.Average = {
-      Absences: [],
-      Tardies: []
-    };
-    _.forEach(results, function(school) {
-      var cumulativeAbsencesData = [];
-      var averageAbsencesData = [];
-      var cumulativeTardiesData = [];
-      var averageTardiesData = [];
-
-      school.data = _.sortBy(school.data, function(record) {
-        return record.date;
-      });
-
-      _.forEach(school.data, function(record) {
-        var dt = new Date(record.date);
-        var utcDt = Date.UTC(dt.getFullYear(), dt.getMonth(), dt.getDate());
-        cumulativeAbsencesData.push([utcDt, record.absences]);
-        averageAbsencesData.push([utcDt, record.absences / record.count]);
-        cumulativeTardiesData.push([utcDt, record.tardies]);
-        averageTardiesData.push([utcDt, record.tardies / record.count]);
-      });
-
-      $scope.Cumulative.Absences.push({
-        name: school._id.name,
-        data: cumulativeAbsencesData
-      });
-
-      $scope.Average.Absences.push({
-        name: school._id.name,
-        data: averageAbsencesData
-      });
-
-      $scope.Cumulative.Tardies.push({
-        name: school._id.name,
-        data: cumulativeTardiesData
-      });
-
-      $scope.Average.Tardies.push({
-        name: school._id.name,
-        data: averageTardiesData
-      });
-
-      $scope.chartConfig.series = $scope[$scope.statType][$scope.truancyType];
+  Visualization.arcaCa().$promise.then(function(results) {
+    function counter(rec, isCfa) {
+      var key = isCfa ? 'cfa' : 'nonCfa';
+      if(rec.arca) { 
```

Comment:
This needs to be reversed the way you have your aggregation pipeline setup. The pipeline does not exclude students with over 20 absences from returning true for arca so this condition never hits any students that are ca true. because all the ca true students are also arca.